[Encryption] PHPORM-360 Document limitations of encryption with collection inheritance (#2790)
doctrine/mongodb-odm: docs/en/reference/attributes-reference.rst

        public string $name;
    }

> The ``#[Encrypt]`` attribute is can be added to an class with `#[EmbeddedDocument]`_.

Reviewer: ```suggestion
The ``#[Encrypt]`` attribute can be added to a class with `#[EmbeddedDocument]`_.
```

This will encrypt the entire embedded document, in the field that contains it.
Queryable encryption is not supported for embedded documents, so the ``queryType``

> argument is not applicable in this case. In this case, the embedded document is

Reviewer: ```suggestion
argument is not applicable. Encrypted embedded documents are 
```

This phrasing feels clearer, as "in this case" implies a different way of using the encryption feature (to me at least).
